lib/resources/r4/coverage.yaml

```diff
@@ -29,7 +27,6 @@ fields:
 
 - name: subscriber
   required: 'Yes'
```

Comment:
double check subscriber and relationship

Reply:
[Double checked.](https://github.com/cerner/fhir.cerner.com/pull/374/commits/6fb0b4a5a1bdcd0b07e086f542d44ded83b12430?file-filters%5B%5D=.yaml#diff-af9c966415db83d073f1bcec5ed6f875L73) I believe subscriber is required, but relationship is not required.